docs/reference/cluster/cluster-info.asciidoc

```diff
@@ -126,17 +131,130 @@ Cumulative size in bytes of all requests from this client.
 ======
 
 
+[[cluster-info-api-response-body-ingest]]
+`ingest`::
+(object)
+Contains ingest information for the cluster.
++
+.Properties of `ingest`
+[%collapsible%open]
+======
+`total`::
+(object)
+Contains information about ingest operations for the cluster.
++
+.Properties of `total`
+[%collapsible%open]
+=======
+`count`::
+(integer)
+Total number of documents ingested during the lifetime of the cluster.
```

Comment:
Is it the lifetime of the cluster? Or the lifetimes of all the nodes that are currently in the cluster (i.e. a node loses these counts on restart right?)?

Reply:
Right... Will see how to reword this... Anyway, this is going to be a really weird number, because it won't be all the time increasing with nodes restarting... 
@dakrone This is an important point to consider for these statistics.

Reply:
Indeed, this is tricky. Each node tracks the information it has seen since it started. This cluster-wide info view is the sum of those since-the-node-started statistics for all the nodes.

It's straightforward enough to talk about on this PR, but I'm not sure what I think would be appropriate wording for documentation. 😬